cmd/root.go

```diff
@@ -50,6 +51,7 @@ func Execute() {
 func init() {
 	cobra.OnInitialize(initConfig)
 	RootCmd.PersistentFlags().StringVarP(&s3Region, "region", "r", "", "s3 region")
+	RootCmd.PersistentFlags().StringArrayVar(&append, "append", []string{}, "variable suffix, e.g: --addto \"DATABASE_URL:?pool=10\"")
```

Comment:
I'd like to avoid to add such global flags if not needed. Example:

```
/treasury import -h
...

Usage:
  treasury import ENVIRONMENT/APPLICATION/ secrets/file/path [flags]

Global Flags:
      --append stringArray   variable suffix, e.g: --addto "DATABASE_URL:?pool=10"

/treasury list -h
List the secrets set for a path

Usage:
  treasury list PATH [flags]

...

Global Flags:
      --append stringArray   variable suffix, e.g: --addto "DATABASE_URL:?pool=10"
```

The append flag is not used in the `import` or `list` commands.